Compiler/General.py

```diff
@@ -363,6 +363,95 @@ def get_if_equal_to_0_code(inside_if_code, offset_to_temp_cell):
     return code
 
 
+def get_bitwise_code(code_logic):
```

Comment:
can you please add documentation at the beginning of this function about what code_logic:
what it assumes - where the pointer is, what is the near cells
what it should do - (from what i understand - move desired bit to desired cell?)
where should pointer be after code_logic ends